app/code/Magento/Customer/Block/Form/Register.php

```diff
@@ -16,6 +16,8 @@
  */
 class Register extends \Magento\Directory\Block\Data
 {
+    private const XML_PATH_NEWSLETTER_ACTIVE = 'newsletter/general/active';
```

Comment:
The reason for adding a constant is mostly to easily reuse the config path in the code base, so I think it should be public. It should be used to replace all the `newsletter/general/active` occurrences in PHP code. And probably the newsletter observer would be a better place to put this constant to, as it is closer to newsletter than to customer module